conf/rubocop_without_rspec.yml

```diff
@@ -134,10 +134,6 @@ Style/StringLiterals:
   Exclude:
     - 'spec/**/*'
 
-# We don't require %i() for an array of symbols.
-Style/SymbolArray:
```

Comment:
Is the `Style/SymbolArray` cop still disabled with this change?

Reply:
#29 added:

```
Style/SymbolArray:
  EnforcedStyle: brackets
```
which silently re-enabled this cop and enforces brackets. Only with this new version of rubocop does it warn of duplicated keys: https://github.com/rubocop-hq/rubocop/commit/4a0a313f9130da6e60e471a28433296706d6332c

All of that is to say this shouldn't change behavior.